index.html

```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Css Component Library</title>
+    <link href="style.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="functionality.js" type = "text/javascript"></script>
+</head>
+
+<body>
+    <div class="header" id='header'>
+        <h1>Css Component Library</h1>
+    </div>
+    <div class='body-container'>
+        <div class='alert-container' id='alerts-id'>
+            <h1>Alerts</h1>
+            <div class="alert alert-success">
+                Success alert
+            </div>
+            <div class="alert alert-failure">
+                Failure alert
+            </div>
+            <div class="alert alert-warning">
```

Comment:
Increase contrast of alerts besides alert-info, which has correct contrast. Check using chrome dev tools.